src/datachain/lib/dc.py

```diff
@@ -484,10 +484,15 @@ def from_dataset(
         fallback_to_remote: bool = True,
```

Comment:
I thought we were trying to avoid "remote" terminology.

Reply:
yes, agree, probably better to use `studio` ?